specification/_json_spec/open_point_in_time.json

```diff
@@ -47,6 +47,10 @@
         "type": "string",
         "description": "Specific the time to live for the point in time",
         "required": true
+      },
+      "allow_partial_search_results": {
+        "type": "boolean",
+        "description": "Specifiy whether to tolerate shards missing from the point in time creation, or throw an exception if any shard is missing. (default: false)"
```

Comment:
Was about to setup a new PR for that; but good to know that automation will take care of this! Should I remove this part to avoid potential conflicts?

Reply:
Also, just to be on the safe side, would automation also handle backports if needed?

Reply:
Yes, the automation handles backports, and your change won't cause a conflict unless the content is different, no worries. I do think it's good to have here.